src/js/helpers/ResourcesHelpers.js

```diff
@@ -153,7 +153,7 @@ export function migrateOldCheckingResourceData(projectDir, toolName) {
  * @param {string} projectDir - path to the project directory
  * @param {function} dispatch - dispatch function
  */
-export function copyGroupDataToProject(gatewayLanguage, toolName, projectDir, dispatch) {
+export function copyGroupDataToProject(gatewayLanguage, toolName, projectDir, dispatch = () => {}) {
```

Comment:
I think we don't want to make the dispatch parameter = to an empty arrow function. I think we benefit more without it because we will be notified when it is undefined. Imagine having to debug this function trying to figure out that you forgot to pass the dispatch argument.